include/seqan3/io/alignment_file/detail.hpp

```diff
@@ -574,37 +579,37 @@ template <TupleLike alignment_type>
     requires std::tuple_size_v<remove_cvref_t<alignment_type>> == 2 &&
              detail::all_satisfy_aligned_seq<detail::tuple_type_list_t<alignment_type>>
 //!\endcond
-void alignment_from_cigar(alignment_type & alignment, std::vector<std::pair<char, size_t>> const & cigar)
+void alignment_from_cigar(alignment_type & alignment, std::vector<cigar> const & cigar_vector)
```

Comment:
Please adapt doxygen's `\param[in]` statement to the renamed cigar_vector parameter.